machine/v1/types_nutanixprovider.go

```diff
@@ -0,0 +1,244 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/klog/v2"
+)
+
+const (
+	// NutanixCredentialsSecretName is the name of the secret holding the credentials for PC client
+	NutanixCredentialsSecretName = "nutanix-credentials"
+)
```

Comment:
Does this need to be here? I wouldn't expect this to be here based on what we have in other platforms. What's the plan for use of this const?

Reply:
Will remove this

Reply:
Removed.